### ISSUE TITLE
Unclear use of the word "respectively"

### ISSUE THREAD

**Issue by getaaron** [jenetics/jenetics.io#17]
The site [uses the word *respectively* several times](https://github.com/jenetics/jenetics.io/search?q=respectively&unscoped_q=respectively) in an unclear way. A few examples:

> Jenetics is an advanced Genetic Algorithm, Evolutionary Algorithm and Genetic Programming library, respectively, written in modern day Java. 

> Jenetics is an advanced Genetic Algorithm, respectively an Evolutionary Algorithm, library written in modern day Java.

I don't understand what the word *respectively* means in this context. If someone can explain it I might be able to open a pull request with a clearer explanation.

**Comment by jenetics** [comment]
What I'm trying to say is, that Jenetics is a Genetic Algorithm, Evolutionary Algorithm and a Genetic Programming library. It's possible that I'm using the word _respectively_ not correctly. The German _beziehungsweise_ would fit perfectly in this context ;-)
If you have a suggestion for rephrasing this sentences, please create a PR.

**Comment by getchar** [comment]
It seems like "beziehungsweise" here is being used to indicate a clarification or restatement, for example, "Er war mit ihm bekannt bzw. befreundet." (Example found [here](https://german.stackexchange.com/questions/9894/when-to-use-beziehungsweise), and incidentally noted as _stilistisch unschön_.)

Although "respectively" seems to be the most common single-word translation of "beziehungsweise," it does not have this meaning in English. We would typically use a phrase:

... that is ...
... that is to say ...
... or better still ...
... more precisely ...
... one/I should say ...
... specifically ...
... in particular ...

In any case, the point is to call the reader's attention to the fact that you're saying the same thing in a different way, either emphasizing a comparison or a distinction which they would not otherwise have noticed. If you don't want to make such an emphasis, the word is best dropped entirely. (This would be my advice regarding the quoted instance.) Overuse of "beziehungsweise" is a German verbal tic that does not translate cleanly into English.

(I'm a native English speaker who's lived in Germany for 5 years. I was familiar with the word "beziehungsweise" but I had to look it up to make sense of this post.)

**tl;dr: If you want to emphasize the fact that you are restating something, use a phrase like "that is," otherwise, "beziehungsweise" can probably be dropped when translating into English.**

**Comment by jenetics** [comment]
Fixed
